client/my-sites/patterns/components/get-started/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { Button } from '@wordpress/components';
+import ImgBlockPatterns from 'calypso/my-sites/patterns/components/get-started/images/block-patterns.svg';
+import ImgHomePage from 'calypso/my-sites/patterns/components/get-started/images/home-page.svg';
+import ImgPageLayouts from 'calypso/my-sites/patterns/components/get-started/images/page-layouts.svg';
+import { PatternsSection } from 'calypso/my-sites/patterns/components/section';
+
+import './style.scss';
+
+export const PatternsGetStarted = () => {
+	return (
+		<PatternsSection
+			title="Get started with patterns"
+			description="Our favorite how-to guides to get you started with patterns."
+			theme="dark"
+			bodyFullWidth
+		>
+			<div className="patterns-get-started--body">
+				<div className="patterns-get-started--body-buttons">
+					<Button className="patterns-get-started--body-buttons-build" href="#">
+						Build a site
+					</Button>
+				</div>
+
+				<div className="patterns-get-started--body-list">
+					<div className="patterns-get-started--body-list-inner">
+						<Button className="patterns-get-started--body-list-item" href="#">
+							<img
+								src={ ImgBlockPatterns }
+								alt="VIDEO TUTORIAL"
+								className="patterns-get-started--body-list-item-image"
+							/>
+							<div className="patterns-get-started--body-list-item-name">VIDEO TUTORIAL</div>
```

Comment:
```suggestion
							<div className="patterns-get-started__item-name">VIDEO TUTORIAL</div>
```
Kind of a nitpick, but I don't think every class name needs to reflect the whole element tree. `body` is definitely superfluous at this level IMO, and perhaps `list` is as well…

I'll leave this to you, but this is a BEM principle I've championed for a long time: class names don't have to reflect the entire DOM structure.

If we decide to pursue this, we should also look for other places to apply it.

Reply:
Yeah, totally agree 👍 Changes and adjusted other components.